src/meta/greedy_load_balancer.cpp

```diff
@@ -453,7 +453,7 @@ bool greedy_load_balancer::copy_primary_per_app(const std::shared_ptr<app_state>
     return true;
 }
 
-bool greedy_load_balancer::copy_secondary_per_app(const std::shared_ptr<app_state> &app)
+bool greedy_load_balancer::copy_secondary(const std::shared_ptr<app_state> &app, bool place_holder)
```

Comment:
What is the `place_holder` used for?

Reply:
The callback in `execute_balance` need two parameters. So this function needs two parameters also, and the secondary one is useless